developer_docs.md

```diff
@@ -61,6 +61,8 @@ You can use either `pyenv` or `python3-venv`:
 
 ##### pyenv
 
+(For Mac OS users.)
```

Comment:
Does `python3-venv` not work on MacOS? The top answer to https://stackoverflow.com/questions/55482197/how-to-correctly-install-python3-and-virtualenv-on-macos-mojave suggests that it does.

Reply:
Ah, yes so this was the result of attempting to walk Stu through this section and the instructions for `python3-venv` not working. But a bit of googling now suggests that it wasn't that `python3-venv` doesn't work for MacOS, it's that `apt-get` doesn't so the instructions as currently written for that tool don't work outside of Linux. I'll try and adjust this PR to reflect this.